src/avram/queryable.cr

```diff
@@ -214,23 +214,29 @@ module Avram::Queryable(T)
   end
 
   def any? : Bool
-    queryable = clone
-    new_query = queryable.query.limit(1).select("1 AS one")
-    results = database.query_one?(new_query.statement, args: new_query.args, queryable: schema_class.name, as: Int32)
-    !results.nil?
+    cache_store.fetch(cache_key, as: Bool) do
+      queryable = clone
+      new_query = queryable.query.limit(1).select("1 AS one")
```

Comment:
No clue how to test the cache on this method since it would require re-opening a method within this block, but which one? Same for the `select_count` which uses different methods...